examples/.interop/next-prisma-starter-websockets/package.json

```diff
@@ -47,19 +50,18 @@
     "@trpc/server": "^10.0.0-proxy-beta.8",
     "clsx": "^1.1.1",
     "next": "^12.3.1",
-    "next-auth": "^4.10.3",
+    "next-auth": "~4.10.3",
```

Comment:
next-auth@^4.11.0 requires next@12.2.5 as peerDep so locking this at 4.10 for now